public/app/features/explore/Explore.tsx

```diff
@@ -225,12 +219,12 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
 
   renderLogsPanel(width: number) {
     const { exploreId, syncedTimes, theme } = this.props;
-
+    const spacing = parseInt(theme.spacing(1).slice(0, -2), 10);
```

Comment:
```suggestion
    const spacing = parseInt(theme.spacing(2).slice(0, -2), 10);
```

Reply:
Thank you @ivanahuckova! I thought i changed it xD